source/request-as-event-emitter.js

```diff
@@ -226,16 +226,6 @@ module.exports = (options, input) => {
 	emitter.retry = error => {
 		let backoff;
 
-		if (is.plainObject(error)) {
-			// Retry with updated options
-			options = {
-				...options,
-				...error
-			};
-
-			error = null;
-		}
-
```

Comment:
Why is this being removed?

Reply:
This code was used for retrying when `afterResponse` returned a plain object as the updated options.